sdks/wasm/src/library_mono.js

```diff
@@ -94,7 +94,7 @@ var MonoSupportLib = {
 			if (!this.mono_wasm_setup_single_step)
 				this.mono_wasm_setup_single_step = Module.cwrap ("mono_wasm_setup_single_step", null, [ 'number']);
 
-			this.mono_wasm_setup_single_step (kind);
+			return this.mono_wasm_setup_single_step (kind);
```

Comment:
It might be good to document briefly somewhere what this return value means and why it's important. I can kind of guess from reading all of the relevant code in this PR but I feel like it would be more enigmatic to someone approaching it without this context. I'm also still not COMPLETELY certain that it means what I believe it means :)